# Fix the hardcoded BLAS library to make use of flexiblas
--- geopsypack-src-3.4.2/ArrayCore/configure-3.4.o	2021-07-30 15:50:19.000000000 +0200
+++ geopsypack-src-3.4.2/ArrayCore/configure-3.4	2023-05-23 16:45:31.000000000 +0200
@@ -557,7 +557,7 @@
 echo "* Dependency lib names [platform dependent]..."
 case "$PLATFORM" in
   "LINUX" | "SUNOS")
-    LIBS="-lQGpCoreTools -lQGpCoreMath -lGeopsyCore -lQGpCoreWave -lDinverCore -lDinverDCCore -lQGpCompatibility -llapack -lblas -lQGpCoreStat "
+    LIBS="-lQGpCoreTools -lQGpCoreMath -lGeopsyCore -lQGpCoreWave -lDinverCore -lDinverDCCore -lQGpCompatibility -lflexiblas -lQGpCoreStat "
   ;;
   "MINGW")
     LIBS="-lQGpCoreTools2 -lQGpCoreMath1 -lGeopsyCore4 -lQGpCoreWave2 -lDinverCore1 -lDinverDCCore2 -lQGpCompatibility1 -llapack -lblas -lQGpCoreStat1 "
@@ -572,7 +572,7 @@
 ###############################################################
 echo "* Dependency lib include paths..."
 cd $BASE_SOURCE_DIR
-for LLIB in -lQGpCoreTools -lQGpCoreMath -lGeopsyCore -lQGpCoreWave -lDinverCore -lDinverDCCore -lQGpCompatibility -llapack -lblas -lQGpCoreStat; do
+for LLIB in -lQGpCoreTools -lQGpCoreMath -lGeopsyCore -lQGpCoreWave -lDinverCore -lDinverDCCore -lQGpCompatibility -lflexiblas -lQGpCoreStat; do
   LIB_BASENAME=$(echo $LLIB | sed "s/^-l//g")
   LIB_PATH_FOUND="no"
   for LIB_PATH in $(find . -name "${LIB_BASENAME}" | sed "s/^\.\///"); do

--- geopsypack-src-3.4.2/QGpCoreMath/configure-3.4.o	2021-07-30 15:50:19.000000000 +0200
+++ geopsypack-src-3.4.2/QGpCoreMath/configure-3.4    2023-05-23 14:35:47.000000000 +0200
@@ -557,7 +557,7 @@
 echo "* Dependency lib names [platform dependent]..."
 case "$PLATFORM" in
   "LINUX" | "SUNOS")
-    LIBS="-lQGpCoreTools -llapack -lblas "
+    LIBS="-lQGpCoreTools -lflexiblas "
   ;;
   "MINGW")
     LIBS="-lQGpCoreTools2 -llapack -lblas "
@@ -572,7 +572,7 @@
 ###############################################################
 echo "* Dependency lib include paths..."
 cd $BASE_SOURCE_DIR
-for LLIB in -lQGpCoreTools -llapack -lblas; do
+for LLIB in -lQGpCoreTools -lflexiblas; do
   LIB_BASENAME=$(echo $LLIB | sed "s/^-l//g")
   LIB_PATH_FOUND="no"
   for LIB_PATH in $(find . -name "${LIB_BASENAME}" | sed "s/^\.\///"); do

--- geopsypack-src-3.4.2/QGpCoreTools/configure-3.4.o	2023-09-14 10:33:58.000000000 +0200
+++ geopsypack-src-3.4.2/QGpCoreTools/configure-3.4	2023-09-14 10:38:55.000000000 +0200
@@ -619,7 +619,7 @@
 
   GCC_OPTIONS="$(for p in $EXTINCDIR; do echo -I $p; done)  $(for p in $EXTLIBDIR; do echo -L $p; done)"
 
-  for OPEN_BLAS_INCLUDE in cblas openblas/cblas cblas-openblas; do
+  for OPEN_BLAS_INCLUDE in flexiblas; do
     echo "*   testing include <${OPEN_BLAS_INCLUDE}.h>..."
     cd $ORIG_DIR
     rm -rf $TEMP_DIR/OpenBLAS
